src/app/forms/suggestion-box/terra-suggestion-box.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="select-box-wrapper"
+     [ngClass]="{
+                  'input-unit-v1': _hasLabel,
+                  'error': !isValid,
+                  'disabled': isDisabled,
+                  'open': '_toggleOpen'
+                }">
+    <input
+        tooltipPlacement="{{inputTooltipPlacement}}"
+        tooltip="{{inputTooltipText}}"
+        tooltipEnable="{{inputTooltipText}}"
+        (focus)="onFocus()"
+        (input)="onChange()"
+        name="{{inputName}}"
+        [(ngModel)]="_currentValue.caption"
+        value={{_selectedValue.caption}}
+    >
```

Comment:
Wir haben unsere eigenen Components, also sollten wir sie auch nutzen. Da es aber dringend ist, mach einen ToDo dazu und wir nehmen das mit in den nächsten Sprint